sample-apps/02-otel-collector/kubernetes/01-configmap.yaml

```diff
@@ -5,20 +5,55 @@ metadata:
   namespace: otel-collector
 data:
   otel-collector-config.yml: |
+    extensions:
+      sigv4auth:
+        region: "__replace-region__"
+        service: "osis"
+
     receivers:
       otlp:
         protocols:
           grpc:
             endpoint: 0.0.0.0:55680
+          http:
+            endpoint: 0.0.0.0:55681
             
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 80
+        spike_limit_percentage: 30
+      batch:
+        send_batch_size: 10
+        timeout: 5s
+    
     exporters:
-      otlp/data-prepper:
-        endpoint: data-prepper.data-prepper.svc.cluster.local:21890
-        tls:
-          insecure: true
+      otlphttp/traces:
+        traces_endpoint: "https://__REPLACE_WITH_OtelTraces_ENDPOINT__/v1/traces"
+        auth:
+          authenticator: sigv4auth
+        compression: none
+      otlphttp/logs:
+        logs_endpoint: "https://__REPLACE_WITH_OtelTraces_ENDPOINT__/v1/logs"
+        auth:
+          authenticator: sigv4auth
+        compression: none        
+      otlphttp/metrics:
+        metrics_endpoint: "https://__REPLACE_WITH_OtelTraces_ENDPOINT__/v1/metrics"
+        auth:
+          authenticator: sigv4auth
+        compression: none
```

Comment:
This is a point I'm wondering if it makes sense to let the user change it or automate it in the setup. I can say that at least 20-30% of participants have difficulty with this step. Either for some invalid character, removed or added something that shouldn't, etc.During the execution I reinforce this message to the participants several times but even so, intercurrences occur.

On the other hand, I always wanted the user to understand how things work and integrate. But it can generate a vision of something difficult (almost a philosophical discussion).

Reply:
I went back and forth on this and the URLs. I am thinking to keep it as is to demonstrate that both region and URLs need to be provided. We have already simplified/scripted a lot of things.

May be leave it as is and learn from running a few of these workshops, then decide if we better off scripting it.

Reply:
scripted endpoint replacement - will make changes in the workshop instructions.